prow/scripts/lib/utils.sh

```diff
@@ -507,6 +508,11 @@ function utils::post_hook() {
         esac
     done
 
+    if [ "$kyma2" = "false" ]; then
```

Comment:
I couldn't find place where we set this value to true.
I think that in `kyma2-gke-external-registry`, we should set to it true